src/uint/div.rs

```diff
@@ -104,22 +228,7 @@ impl<const LIMBS: usize> Uint<LIMBS> {
     /// When used with a fixed `rhs`, this function is constant-time with respect
     /// to `self`.
     pub const fn rem_vartime(&self, rhs: &NonZero<Self>) -> Self {
-        let mb = rhs.0.bits_vartime();
-        let mut bd = Self::BITS - mb;
-        let mut rem = *self;
-        let mut c = rhs.0.wrapping_shl_vartime(bd);
-
-        loop {
-            let (r, borrow) = rem.sbb(&c, Limb::ZERO);
-            rem = Self::select(&r, &rem, ConstChoice::from_word_mask(borrow.0));
-            if bd == 0 {
-                break;
-            }
-            bd -= 1;
-            c = c.shr1();
-        }
-
-        rem
+        self.div_rem_vartime(rhs).1
```

Comment:
I wonder if manually modifying `div_rem_vartime()` with the assumption that the quotient is discarded produces a faster result.

Reply:
I don't believe there would be much benefit, it would just avoid storing the quotient words and then shifting them at the end. I think I also tried an inlined version to see if the optimizer would catch anything and it didn't seem to have any impact.